charts/argo-cd/Chart.yaml

```diff
@@ -27,4 +27,4 @@ annotations:
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
     - kind: changed
-      description: Bump argo-cd to v2.12.6
+      description: Bump argo-cd to v2.13.0-rc3
```

Comment:
It should be 2.13.0 instead of 2.13.0-rc3 ?

Reply:
Well spotted! :)